week-1-practise/level-3/1-piping.js

```diff
@@ -11,7 +11,11 @@
 */
 
 var startingValue = 5;
+function allInOne(num) {
+  return parseFloat(num * 2 + 10).toFixed(2);
```

Comment:
The exercise was asking to do this using the provided functions: `add`, `multiply` and `format`.